tests/cel-validation/testdata/aigatewayroutes/basic.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: aigateway.envoyproxy.io/v1alpha1
-kind: LLMRoute
+kind: AIGatewayRoute
```

Comment:
not sure how this was not caught by CI in the previous PR

Reply:
yeah super weird...